src/apiServices/dialogAPI.ts

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+import { TypesFileEnum } from "../assets/constants/api.contsnts";
+import { ScrollDataType } from "../interfaces/common";
+import { CreateDialogDataType } from "../interfaces/dialog";
+
+export class DialogAPI {
+    static createDialog = (data: CreateDialogDataType) => axios.post("/dialog", data);
+
+    static getDialogsById = ({ userId, limit, page }: ScrollDataType) =>
+        axios.get(`/dialog?userId=${ userId }&limit=${ limit }&page=${ page }`);
```

Comment:
you can provide query params as object. Bad practice to build query manually

Reply:
Yes, the project isn't small((